composer.json

```diff
@@ -16,7 +16,7 @@
         "ext-json": "*",
         "ext-sockets": "*",
         "php-amqplib/php-amqplib": ">=2.9",
-        "laravel/framework": "^5.6|^6.0",
+        "laravel/framework": "^5.6|^7.0",
```

Comment:
```suggestion
        "laravel/framework": "^5.6|^6.0|^7.0",
```

Reply:
with "^5.6|^7.0" it would work for 6.0 too but lets get this explicit.